packages/ui/src/primitives/tailwind/Select/index.tsx

```diff
@@ -144,10 +144,11 @@ const Select = <T extends OptionValueType>({
         endComponent={
           <MdOutlineKeyboardArrowDown
             size="1.5em"
-            className={`text-theme-primary transition-transform ${showOptions.value ? 'rotate-180' : ''}`}
+            className={`text-[#444444] transition-transform ${showOptions.value ? 'rotate-180' : ''}`}
             onClick={toggleDropdown}
           />
         }
+        containerClassname="rounded-md"
```

Comment:
why did we add border radius here?
[in designs](https://www.figma.com/design/ln2VDACenFEkjVeHkowxyi/iR-Engine-Design-Library-File?node-id=2508-3527&t=FC7FlAaAzLUoVy6o-0), select primitive does not have a border radius

Reply:
oh I didnt realized we had this file for primitive. will resolve